pept/tracking/birmingham_method/birmingham_method.py

```diff
@@ -0,0 +1,231 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pept
+import os
+import matplotlib.lines as mlines
+
+from .extensions.birmingham_method import birmingham_method
+
+class BirminghamMethod():
```

Comment:
The class declaration does not need "()"

Reply:
Done!